operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -53,10 +53,6 @@ type CentralSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Customizations,order=6,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	Customize *CustomizeSpec `json:"customize,omitempty"`
 
-	// Miscellaneous settings.
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Miscellaneous,order=7,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
-	Misc *MiscSpec `json:"misc,omitempty"`
```

Comment:
Never remove a property from a CRD !!! Must be marked as deprecated

Reply:
Thanks! This ticket is parked for now, but when I come back to working on it, I will definitely fix that issue.